WeakAuras/Types.lua

```diff
@@ -2223,4 +2223,168 @@ if WeakAuras.IsClassic() then
   end
 
   WeakAuras.glow_types.ACShine = nil
+
+  -- from https://github.com/tullamods/Dominos/blob/master/Dominos/bars/getActionReagentUses.lua
+  WeakAuras.spell_reagents = {
+    -- Druid
+    -- Gift of the Wild
+    [GetSpellInfo(21849)] = 17021,
```

Comment:
spell data is asynchronously loaded, I thought. So I would expect that in some cases this'd cause a `table index is nil` error, and break things.